### PR TITLE
Adding documentation for publishing to rubygems

### PR DESCRIPTION
**Pull request by line47**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,25 +102,29 @@ You will need to restart your Jekyll server to see the effects.
...
    `Gemfile` to point at the local clone of this repo:
 
     ```ruby
     gem 'uswds-jekyll', :path => '../path/to/uswds-jekyll'
     ```
+### Publish to Rubygems
+1. Update `spec.version = "NUMBER HERE"` in the uswds-jekyll.gemspec file to the version you want to publish
+1. Run `bundle install`
+1. Run `bundle exec rake release`
+1. Add a PR for the update and get it merged
```
Comment (hursey013) [inline README.md]: I'm wondering if this step should be done before `bundle exec rake release`? Presumably you'd want a second pair of eyes on the PR prior to publishing it, but I may not be following the process correctly.

Comment (line47) [reply README.md]: @jeremiak ❤️ to get your thoughts here

Comment (jeremiak) [reply README.md]: Ya, I think @hursey013 is correct. `rake release` will push to RubyGems so probably best to run the first two steps and _then_ open the PR. Following the merge, then run `rake release`.

**Comment by line47** [comment]
Updated and ready for review 👍

**Pull request merged**